Library/Formula/crawl.rb

```diff
@@ -0,0 +1,37 @@
+class Crawl < Formula
+  desc "Dungeon Crawl: Stone Soup"
+  homepage "http://crawl.develz.org/"
+  version "0.17.1"
+
+  url "https://github.com/crawl/crawl.git",
+    :shallow => false,
+    :revision => "942a3d2973debefdaddbf6291e4ebc890725b920"
```

Comment:
Ok, that's fine about submodules. Just remove the `version` if it complains. Thanks!